### PR TITLE
docs(x/precisebank): Add state, keepers, events, client to spec 

### PR DESCRIPTION
**Pull request by drklee3**
## Description
- Include the typical parts of spec, on top of the main math logic

### CODE REVIEW

File: x/precisebank/README.md [hunk 1]
```diff
@@ -354,3 +378,195 @@ $$b'(R) - b(R) = \begin{cases} 0 & r' > r \land f'(1) < f(1) \\
...
 0 & r' \leq r \land f'(1) \geq f(1) \\
 \end{cases}$$
+
+## State
+
+The `x/precisebank` module keeps state of the following:
+1. Account fractional balances.
+2. Remainder amount. This amount represents the fractional amount that is backed
+   by the reserve account but not yet in circulation. This can be non-zero if
+   a fractional amount less than `1ukava` is minted.
```
Comment (nddeluca) [inline x/precisebank/README.md:388]: Should we add a note about current mainnet state and remainder is always zero (we think) at the end of each block due to the EVM always using matching mint/burns?

File: x/precisebank/README.md [hunk 2]
```diff
@@ -354,3 +378,195 @@ $$b'(R) - b(R) = \begin{cases} 0 & r' > r \land f'(1) < f(1) \\
...
+    }
+  ]
+}
+```
+
+#### MintCoins
+
+```json
+{
+  "type": "mint",
```
Comment (nddeluca) [inline x/precisebank/README.md:456]: This is actually `coinbase`?

File: x/precisebank/README.md [hunk 3]
```diff
@@ -354,3 +378,195 @@ $$b'(R) - b(R) = \begin{cases} 0 & r' > r \land f'(1) < f(1) \\
...
+```
+
+## Messages
+
+The `x/precisebank` module does not have any messages and is intended to be used
+by other modules as a replacement of the bank module.
+
+## Events
+
+### Keeper Events
```
Comment (nddeluca) [inline x/precisebank/README.md:416]: We missed the "coin_spent" and "coin_received" events, oops

**Review by nddeluca — APPROVED** [review]
The spec is approved, feel free to merge into the other PR and we can get more eyes on it.

However, a couple things to update -- the `mint` event is called `coinbase`, and we forgot about the `coin_spent` and `coin_received` events in the x/precisebank module, but that's work for another PR and we can add those to the spec then.

File: x/precisebank/README.md (hunk 2 above)
Comment (drklee3) [reply x/precisebank/README.md]: yup, updated

**Pull request merged**